connectors/2.0/_index.md

```diff
@@ -1,31 +1,31 @@
 ---
 title: "Jakarta Connectors 2.0 (milestone)"
-date: 2020-06-12
+date: 2020-09-13
```

Comment:
Since you are making other updates, you might as well put in a more accurate date.  If a vote goes out tomorrow, the soonest that this is available is Oct 30.  Or, you could just match the date on your spec of Oct 10.  Whatever...

Reply:
Done